docs/src/pages/components/data-grid/selection/selection.md

```diff
@@ -28,13 +28,14 @@ Row selection can be performed with a simple mouse click, or using the [keyboard
 ### Single row selection
 
 Single row selection is enable by default with the `DataGrid` component.
+To unselect a row, hold the <kbd class="key">CTRL</kbd> key and click on it.
 For the `XGrid`, you need to disable multiple row selection with `disableMultipleSelection={true}`.
 
 {{"demo": "pages/components/data-grid/selection/SingleRowSelectionGrid.js", "bg": "inline"}}
 
 ### Multiple row selection [<span class="pro"></span>](https://material-ui.com/store/items/material-ui-pro/)
 
-To activate multiple selection, put focus the `XGrid` component and hold the <kbd class="key">CTRL</kbd> key while selecting rows.
+To select multiple rows, focus on the `XGrid` component and hold the <kbd class="key">CTRL</kbd> key while selecting rows.
```

Comment:
Do we need the focus on the grid? I thought we fixed that 🤔

Reply:
Yes we fixed that 👍 . I rephrased it.